app/mailers/email_to_provider_mailer.rb

```diff
@@ -5,7 +5,7 @@ def notify(message_class, submission)
       message = message_class.new(submission.latest_version.application)
       set_template(message.template)
       set_personalisation(**message.contents)
-      mail(to: message.recipient)
+      mail(to: Array(message.recipient))
```

Comment:
maybe instead of just casting we could have a class whose responsibility is to clean up a message for  notify_gov_uk interface expectations - might signal intent/be testable also

Reply:
Everything else matches already AFAICT

Reply:
You could argue that the place to do this is inside the `message.recipient` definition in the common gem. The advantage of that would be that since both the app store and the caseworker app use the common gem you'd only need to make this change once; otherwise you have to do this both here and in Nsm::EmailToProviderMailer and PriorAuthority::EmailToProviderMailer in the caseworker app.

N.B. either way, you also need to make the change in Nsm::SubmissionMailer and PriorAuthority::SubmissionMailer, as they don't use the same `message` system to send their acknowledgement-of-submission emails.

Reply:
That's a good point I wasn't aware of until our discussion yesterday. Okay I'll shelve this for now and include it as part of the "move where we send emails" ticket